Dockerfile.cli

```diff
@@ -4,6 +4,9 @@ WORKDIR /vcluster-dev
 ARG TARGETOS
 ARG TARGETARCH
 
+# Install helm binary
+RUN curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3 && chmod 700 get_helm.sh && ./get_helm.sh
```

Comment:
This is copied from our other Dockerfile - https://github.com/loft-sh/vcluster/blob/f4771604bbe19db52caf34544bad846ff2aabaa9/Dockerfile#L11-L12